gitmanager.py

```diff
@@ -405,8 +414,8 @@ def reject_pull_request(cls, pr_id, comment=""):
             if response:
                 if response.json()["state"] == "closed":
                     git.push('-d', origin_or_auth, ref)
-                    return "Closed pull request [#{0}](https://github.com/{1}/pull/{0}).".format(
-                        pr_id, GlobalVars.bot_repo_slug)
+                    return ("Closed pull request [#{0}](https://github.com/{1}/pull/{0})." +
```

Comment:
Could you shift the punctuation (".") to be after your addition? Not exactly mission-critical, but the [output in chat](https://chat.stackexchange.com/transcript/message/65714945#65714945) has a slight punctuation issue.